propagator/opentelemetry-propagator-jaeger/src/opentelemetry/propagators/jaeger/__init__.py

```diff
@@ -79,9 +79,11 @@ def inject(
         span = trace.get_current_span(context=context)
         span_context = span.get_span_context()
         if span_context == trace.INVALID_SPAN_CONTEXT:
+            print(f'invalid span context')
```

Comment:
We don't use `print` to inform the user of a potential problem, but use a logger with a warning message.

Reply:
Sigh, this is why you shouldn't work late 🤦 . This was just for debugging, I've removed it now.